testnet/launcher/l2contractdeployer/docker.go

```diff
@@ -90,7 +90,7 @@ func (n *ContractDeployer) WaitForFinish() error {
 	defer cli.Close()
 
 	// make sure the container has finished execution
-	err = docker.WaitForContainerToFinish(n.containerID, 3*time.Minute)
+	err = docker.WaitForContainerToFinish(n.containerID, 10*time.Minute)
```

Comment:
3mins was sometimes too quick for sepolia, this script does quite a lot now with a few waits - don't think there's a need to rush it.